keyboards/atlantis/ps17/ps17.c

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 mjbogusz (@mjbogusz)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "quantum.h"
+
+layer_state_t layer_state_set_kb(layer_state_t state) {
+    /* Display current layer using indicator LEDs */
+    writePin(LED_INDICATOR_0_PIN, !IS_LAYER_ON_STATE(state, 1));
+    writePin(LED_INDICATOR_1_PIN, !IS_LAYER_ON_STATE(state, 2));
+    writePin(LED_INDICATOR_2_PIN, !IS_LAYER_ON_STATE(state, 3));
+    return layer_state_set_user(state);
+}
+
+void keyboard_pre_init_kb(void) {
+    /* Set indicator LEDs as outputs */
+    setPinOutput(LED_INDICATOR_0_PIN);
+    setPinOutput(LED_INDICATOR_1_PIN);
+    setPinOutput(LED_INDICATOR_2_PIN);
+    keyboard_pre_init_user();
+}
+
+#if !defined(ENCODER_MAP_ENABLE)
```

Comment:
this should be if the encoder feature is enabled.  If it's turned off, it will cause errors during compilation, otherwise.  Also, if encoder map is enabled, it will optimize this function out of the code, since it won't get called. 

So, you'd just want: 
```suggestion
#if defined(ENCODER_ENABLE)
```

Reply:
The encoder is soldered in in this keypad and is thus enabled in `info.json` - but I think this (`ENCODER_ENABLE`) approach is slightly more clear in the intent and it sets a good example, so: fixed!